modules/heat_conduction/doc/content/source/fvbcs/FVFunctorConvectiveHeatFluxBC.md

```diff
@@ -0,0 +1,20 @@
+# FVFunctorConvectiveHeatFluxBC
+
+!syntax description /FVBCs/FVFunctorConvectiveHeatFluxBC
+
+# Description
+
+This boundary condition computes convective heat flux $q'' = H \cdot (T - T_{inf})$, where $H$ is convective heat transfer coefficient,
+$T$ is the temperature, and $T_{inf}$ is far field temperature. Both $H$ and $T_{inf}$ are coupled as functors.
+The domain can be specified as either a fluid or a solid using $is_solid$. For a solid domain, the equation above is applied.
```

Comment:
The domain of the `variable` can be specified ...(i.e. is the variable the solid or the fluid temperature).